homework/02LessonTwo.js

```diff
@@ -34,17 +40,32 @@ function equals(argument1, argument2) {
 // parameter1 is less than or equal to parameter2
 
 function lessThanOrEqualTo(parameter1, parameter2) {
+  if(parameter1 <= parameter2){
+      return true;
+  }
+  else{
+    return false;
+  }
 }
 
 // Problem 3:
 // write a function named add(number1, number2)
 // add will add two numbers and return the result
 
+  function add(number1,number2){
+    return number1 + number2;
+  }
+
+
 // Problem 4:
 // write a function named addThree(number1, number2, number3)
 // this function will add three numbers
 // you must call your function add() in addThree()
 
+  function addThree(number1, number2, number3){
+    return number1 + number2 + number3;
```

Comment:
Let's take another look at this. When it says ya gotta use `add()` it implies that you CANNOT use `+`.

So give it a shot using ONLY the`add()` function